### ISSUE TITLE
insight view flickering

### ISSUE THREAD

**Issue by shaykeren** [digma-ai/digma-intellij-plugin#323]
2 issues:
* insight view flickering
* flickering randomly ( not on each X sec)
please see my attached video

https://user-images.githubusercontent.com/1392692/220845125-2ccab1ff-af62-4a31-a599-3388f60eaa5d.mov

**Comment by MykolaYaremchuk** [comment]
After discussion with Shalom:

we have 2 ways to go:

1. fix flickering by calculating the height of each label - this approach Shalom already started - but it's quite complicated task and it requires 2-3 more days to finish it. This is big refactoring which can cause a lot of new bugs on UI because it affects all JLabel components which we use.

2. or to change the way how we refresh our insights - execute deep comparison of old insights list and new insights list - and refresh the UI ONLY if we got some different new insights (so usual background flickering will not be visible because 99% of the time we don't have new insights and there is no need to refresh the UI).

2nd approach can be implemented today.
1st approach requires 2-3 days.


so taking into account fact that in few month we are going to drop Swing and inject React app into WebView (the whole sidepane will be written in React and injected into Digma tool window), `**we decided not to waste much time and effort on this flickering issue today and to go with 2nd approach to update the UI ONLY when we have new changes from the API**`.


solution 2 does not work perfect - we still see flickering each minute because `actualStartTime` field is being updated each minute by our backend Job and this is correct flow (Asaf clarified it to me). 
Idea to exclude `actualStartTime` field from comparison also is not so good because then we crash the flow of Recalculate insight process (condition when to display "Applying the new time filter changes" message and when display "Data from 2 minutes" compares actualStartTime and customStartTime). I tried to implement custom comparator to exclude that field - but it was not working fine due to calculate logic.
